### ISSUE TITLE
"fit list" does not list existing fits without peaks (e.g. integrals)

### ISSUE THREAD

**Issue by j-kleemann** [op3/hdtv#1]
HDTV allows to store a fit without any peaks, e.g. by setting region markers (and optionally background markers), executing "fit execute" (or "fit intergral execute") and then "fit store" (or using the -S flag in the previous step.
Altough at first a bit strange, I get that this behaviour is intended to be able to treat and store proper fits and integrals without any actual peaks fitted in the same manner.
A current drawback of this is, however, that stored fits without any peaks are not listed by fit list, while still being counted at its last line. This can be confusing.
Especially it's not straightfoward to find out which fit IDs are used and which are free.

It should be considered to let "fit list" list existing fits without peaks for example as a single line with the peak id being `-` and all other properties being either `--` or the respective integral values.
E.g. for fit ID 2 being non-existent (deleted) and ID 3 being an integral:

```
  id  stat  chi    channel         pos          vol        width
 1.0     V   19   3802.3(1.4)  2841.5(1.0)    1.4(8)e+02  3.6(1.3)
 3.-     I   --           --           --            --         --
 4.0     V   21  3.43(10)e+03  2.56(8)e+03      3(9)e+05  6(6)e+01
 5.0     V  200    675.37(19)   510.99(15)    5.8(4)e+02  4.18(32)
 5.1     V  200      806.5(4)   608.74(31)       170(28)  4.18(32)```
 
4 peaks in 4 fits.

**Comment by op3** [comment]
You can already use `fit integral list` to print a list of the integrals of all fits.

**Comment by j-kleemann** [comment]
Yes I know that, but this is not about printing the integrals, but being able to see which fit IDs are taken and which are not. Having to use the `fit integral list` command when you can get in this situation by never even touching the `fit integral` commands is not obvious.
A new user can get in this situation by setting region markers, forgetting to set peak markers, fitting and storing and will then wonder why hdtv does not use one ID.

**Comment by j-kleemann** [comment]
Implemented in 21b018c32e60feaea803bec0f76ed8dc24f3e7ca

Looks like this at the moment (status `I` means no peaks, so effectively an integral only):
```  
id  stat  chi    channel       pos          vol        width
 0.-    IV
 1.0     V  148  3473.35(14)  1019.94(4)  9.83(23)e+04  6.87(13)
```
